.rubocop.yml

```diff
@@ -7,3 +7,6 @@ AllCops:
 
 ParameterLists:
   CountKeywordArgs: false
+
+Metrics/ModuleLength:
```

Comment:
Looks like this one, https://github.com/CocoaPods/Rainforest/blob/master/Rakefile#L467-L497